sql/core/src/test/resources/sql-tests/inputs/show-views.sql

```diff
@@ -0,0 +1,31 @@
+-- Test data.
+CREATE DATABASE showdb;
+USE showdb;
+CREATE TABLE tbl(a String, b Int, c String, d String) USING parquet;
+CREATE VIEW view_1 AS SELECT * FROM tbl;
+CREATE VIEW view_2 AS SELECT * FROM tbl WHERE c='a';
+CREATE TEMPORARY VIEW view_3(e int) USING parquet;
+CREATE GLOBAL TEMP VIEW view_4 AS SELECT 1 as col1;
```

Comment:
BTW, please switch line 7 and 8. In this PR, global temp view (`view_4`) is printed out before local temp view (`view_3`).